doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_hdfs_sink.md

```diff
@@ -268,6 +268,7 @@ NOTE: `hive` is the Hive CLI for locally querying the data.
 | Parameter | Mandatory | Default value | Comments |
 |---|---|---|---|
 | type | yes | N/A | Must be <i>com.telefonica.iot.cygnus.sinks.NGSIHDFSSink</i> |
+| attr_metadata_store | no | true | If true, it will store metadata as usual. If false it will not store metadata regardless of the `file_format` type  |
```

Comment:
This has been included in this PR? Or it's a gap (i.e. something developed in aprevious PR but not documented until now)?

Reply:
Just moved it a little bit up on the list.

Reply:
Ah, ok, I didn't realize the deleted L302 line.

NTC